code/game/objects/structures/inflatable.dm

```diff
@@ -7,23 +7,36 @@
 	starting_materials = list(MAT_PLASTIC = 1.5*CC_PER_SHEET_MISC)
 
 	var/deploy_path = null
+	var/tmp/inflating = FALSE
 
 /obj/item/inflatable/attack_self(mob/user)
 	if(!deploy_path)
 		return
 	if(!istype(user.loc, /turf))
 		return
 
+	add_fingerprint(user)
 	if(user.drop_item(src))
-		inflate(user)
+		inflating = TRUE
+		anchored = 1
+		to_chat(user, "<span class='notice'>You pull the inflation cord on \the [src].</span>")
+		spawn(10)
```

Comment:
While I think that's pretty damn cool that's not really addressing @Probe1's complaints regarding balance